llvm/lib/Target/NVPTX/NVPTXISelLowering.cpp

```diff
@@ -436,6 +436,15 @@ NVPTXTargetLowering::NVPTXTargetLowering(const NVPTXTargetMachine &TM,
       case ISD::FADD:
       case ISD::FMUL:
       case ISD::FSUB:
+      case ISD::SELECT:
+      case ISD::SELECT_CC:
```

Comment:
Wouldn't select always be supported? Although I don't see this function called for SELECT so I guess it does matter but it should probably be removed?

Reply:
According to PTX ISA,
setp.{bf16/bf16x2} requires sm_90 or higher.
set.{u16, u32, s16, s32}.bf16, set.{u32, s32, bf16x2}.bf16x2, set.bf16.{s16,u16,f16,b16,s32,u32,f32,b32,s64,u64,f64,b64} require sm_90 or higher.